geode-core/src/integrationTest/java/org/apache/geode/cache30/ShorteningExpirationTimeRegressionTest.java

```diff
@@ -53,67 +48,52 @@
   private static final int LONG_WAIT_MS = 2 * 60 * 1000;
   private static final int SHORT_WAIT_MS = 1;
 
-  private static final String KEY = "key";
```

Comment:
What was the reason for dropping this symbolic constant? That seems better than have the literal "key" used in multiple places.